easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-11.3.0.eb

```
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'UDA'
+version = '2.6.0'
+
+homepage = 'https://github.com/ukaea/UDA'
+description = """Universal Data Access"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
```

we already have `Boost` at `GCC` level, so we should really use that (and bump this easyconfig to `GCC`)
```suggestion
toolchain = {'name': 'GCC', 'version': '11.3.0'}
```